### DISCUSSION TITLE
Add to cart without ajax

### DISCUSSION THREAD

**Discussion by iPolska** [bootscore/bootscore#270] [Q&A] [answered]
Is there any way to handle adding to cart without ajax?

**Comment by crftwrk** [comment, marked as answer]
Not in a simple way because bootScore's focus is on ajax cart. We're refactor the ajax cart in a further release https://github.com/bootscore/bootscore/issues/215. Then it is possible to disable ajax with a simple function in child. But for now I suggest roughly delete functions in main-theme, of course you have to do this again when updating the theme.

#### 1. Remove ajax cart function

Delete snippet https://github.com/bootscore/bootscore/blob/main/woocommerce/woocommerce-functions.php line `29`

```php
// Register Ajax Cart
if (!function_exists('register_ajax_cart')) :

  function register_ajax_cart() {
    require_once('ajax-cart/ajax-add-to-cart.php');
  }
  add_action('after_setup_theme', 'register_ajax_cart');

endif;
// Register Ajax Cart End
```

**OR**

Place the empty function in your child's `functions.php`

```php
function register_ajax_cart() {
}
add_action('after_setup_theme', 'register_ajax_cart');
``` 

#### 2. Remove offcanvas open on click `add_to_cart_button`

Delete snippet in https://github.com/bootscore/bootscore/blob/main/woocommerce/js/woocommerce.js line `3`

```JS
$('.single_add_to_cart_button:not(.product_type_variable):not(.product_type_external):not(.product_type_grouped)').attr('data-bs-toggle', 'offcanvas').attr('data-bs-target', '#offcanvas-cart');
```

#### 3. Make `wc-forward` button visible

Add this to your child

```CSS
.alert .button.wc-forward, 
.must-log-in .button.wc-forward, 
.wc-stripe-error .button.wc-forward {
  display: block;
}
```

#### 4. Disable WC ajax in backend

Uncheck `Enable AJAX add to cart buttons on archives` in backend WooCommerce > Settings > Products

Does this help?
